eclair-core/src/test/scala/fr/acinq/eclair/blockchain/fee/BitcoinCoreFeeProviderSpec.scala

```diff
@@ -90,7 +85,33 @@ class BitcoinCoreFeeProviderSpec extends TestKitBaseClass with BitcoindService w
       blocks_1008 = fees(1008)
     )
 
-    val mockBitcoinClient = new BasicBitcoinJsonRPCClient(rpcAuthMethod = UserPassword("", ""), host = "localhost", port = 0) {
+    val mockBitcoinClient = createMockBitcoinClient(fees, ref.mempoolMinFee)
+
+    val mockProvider = BitcoinCoreFeeProvider(mockBitcoinClient, FeeratesPerKB(FeeratePerKB(1 sat), FeeratePerKB(1 sat), FeeratePerKB(2 sat), FeeratePerKB(3 sat), FeeratePerKB(4 sat), FeeratePerKB(5 sat), FeeratePerKB(6 sat), FeeratePerKB(7 sat), FeeratePerKB(8 sat)))
+    mockProvider.getFeerates.pipeTo(sender.ref)
+    assert(sender.expectMsgType[FeeratesPerKB] == ref)
+  }
+
+  test("get fee rates when no fee rate is found") {
```

Comment:
I don't see the point of this additional test to be honest. It doesn't test a behavior we actually rely on and will break whenever `bitcoind` changes their error message...

Reply:
OK. I removed it.